rlclientlib/live_model_impl.cc

```diff
@@ -208,11 +208,15 @@ namespace reinforcement_learning {
     RETURN_IF_FAIL(check_null_or_empty(event_id, _trace_logger.get(), status));
     RETURN_IF_FAIL(check_null_or_empty(context_json, _trace_logger.get(), status));
 
+    utility::ContextInfo context_info;
+    RETURN_IF_FAIL(utility::get_context_info(context_json, context_info, _trace_logger.get(), status));
+
     // Ensure multi comes before slots, this is a current limitation of the parser.
-    RETURN_IF_FAIL(utility::validate_multi_before_slots(context_json, _trace_logger.get(), status));
```

Comment:
are we still using these functions? do we want to clean their implementation up as well?